t/25-cache-service.t

```diff
@@ -141,6 +140,12 @@ sub test_download {
     ok($asset_request->minion_id, "Minion job id recorded in the request object") or die diag explain $asset_request;
 }
 
+sub perform_job_in_foreground {
```

Comment:
Would it be possible to use `Minion::perform_jobs_in_foreground` here?

Reply:
Since the cache service client is stateful, with Minion job ids as metadata, this is one of the cases where i prefer not to use `perform_jobs_in_foreground`.